ansible/group_vars/store.yml

```diff
@@ -2,6 +2,7 @@
 # Custom SSH accounts, should start from UID 8000.
 bootstrap__active_extra_users:
   - { name: ivan,    uid: 8000, groups: ['docker', 'dockremap'], key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJBdm8y1PfWjT1pioaWJSZ2ETrUySb+dS/ifDg+VIpLY ivansete@status.im' }
+bootstrap__logrotate_frequency: 'hourly'
```

Comment:
```suggestion

# Comment explaining why
bootstrap__logrotate_frequency: 'hourly'
```

Reply:
Comment added to `ansible/group_vars/store.yml` and `ansible/group_vars/boot.yml`